### ISSUE TITLE
Insert/edit image/media dialogs not populated with CP-centric items

### ISSUE THREAD

**Issue by johnalarcon** [ClassicPress-research/ClassicPress-Editor#4]
Relevant buttons...
![image](https://user-images.githubusercontent.com/12477319/131266364-cec00497-8695-4c70-876a-73a62a2eae54.png)

These buttons do work to insert images/media, however, they are not populated with ClassicPress content. These buttons are not  necessarily needed, though, as there is already an existing "Add Media" button (also in the screenshot) which serves the same purpose and works as expected.

**Comment by simplycomputing** [comment]
It doesn't quite work as expected. You get different looking options for dealing with the image which will end up confusing users. For example:
![image1](https://user-images.githubusercontent.com/46998578/131279828-1a06a244-659e-41cf-936d-ae240f712efb.jpg)
![image2](https://user-images.githubusercontent.com/46998578/131279837-9d0c644c-75a7-4c75-ba24-31dedcf93576.jpg)

**Comment by viktorix** [comment]
I agree, buttons should be removed from the editor. Add Media button is the way to go.

**Comment by joyously** [comment]
Please test with the latest version and close this issue if fixed.

**Comment by johnalarcon** [comment]
I've had a couple less-than-productive days while setting up a new system, but, will be testing this today.

**Comment by johnalarcon** [comment]
Actually, I just did a quick test on this while setting up a post for another test. :) The `Add Media` button does insert an image as expected. However, when you select the image in the editor, it doesn't have any way to pop open the media overlay – the "quickbars" are missing. 

New Editor

![image](https://user-images.githubusercontent.com/12477319/132924058-22fe9d77-aad5-4b9c-9e62-ccb2b83a35ac.png)

Old Editor

![image](https://user-images.githubusercontent.com/12477319/132924142-0d0d04b8-ab0d-40d4-af5f-1f1db8f15198.png)

**Comment by joyously** [comment]
This issue is referencing two different things. The issue was written about the generic media buttons that were on the toolbar, and were removed. You are now testing the core Add Media button, which is not what the issue is.
The wpeditimage plugin is for editing images. It is being loaded, but I think it's the wordpress plugin that connects the two. The wordpress plugin is not loaded currently, and I commented out a chunk of it that creates a floating toolbar (likely for the image edit buttons).
